sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/indexes/implementation/models/AnalyzeRequest.java

```diff
@@ -25,31 +26,32 @@
     private String text;
 
     /*
-     * The name of the analyzer to use to break the given text. If this
-     * parameter is not specified, you must specify a tokenizer instead. The
-     * tokenizer and analyzer parameters are mutually exclusive.
+     * The name of the analyzer to use to break the given text.
      */
     @JsonProperty(value = "analyzer")
     private LexicalAnalyzerName analyzer;
 
     /*
-     * The name of the tokenizer to use to break the given text. If this
-     * parameter is not specified, you must specify an analyzer instead. The
-     * tokenizer and analyzer parameters are mutually exclusive.
+     * The name of the tokenizer to use to break the given text.
      */
     @JsonProperty(value = "tokenizer")
     private LexicalTokenizerName tokenizer;
 
+    /*
+     * The name of the normalizer to use to normalize the given text.
+     */
+    @JsonProperty(value = "normalizer")
+    private LexicalNormalizerName normalizer;
+
     /*
      * An optional list of token filters to use when breaking the given text.
-     * This parameter can only be set when using the tokenizer parameter.
      */
     @JsonProperty(value = "tokenFilters")
     private List<TokenFilterName> tokenFilters;
 
     /*
      * An optional list of character filters to use when breaking the given
-     * text. This parameter can only be set when using the tokenizer parameter.
+     * text.
      */
     @JsonProperty(value = "charFilters")
     private List<CharFilterName> charFilters;
```

Comment:
Is this parameter now ignored by the service if set when tokenizer parameter is not set? Did it throw earlier? Curious why the semantics changed?

Reply:
I believe this has changed to work with both `tokenizer` and `normalizer`